src/lib/logstore/log_dev.hpp

```diff
@@ -136,6 +136,7 @@ struct log_group_header {
     uint32_t footer_offset;      // offset of where footer starts
     crc32_t prev_grp_crc;        // Checksum of the previous group that was written
     crc32_t cur_grp_crc;         // Checksum of the current group record
+    logdev_id_t logdev_id;       // Logdev id
```

Comment:
because we have multiple logdev and if we are debugging log entries, we dont know which logdev its part of. just for debugging.

Reply:
It will increase the size of each entry persisted. Is there any other way possible to this information without adding it here?

Reply:
Yeah currently dont see any other easy option. Since its group header, hopefully it less size compared to each log entry.

Reply:
Can you please create a github issue to remind this. When we tackle this next we can take that up.

Reply:
https://github.com/eBay/HomeStore/issues/346